From: Marc Leeman <marc.leeman@gmail.com>
Date: Mon, 8 Jul 2024 09:53:51 +0100
Subject: Do not install LICENSE file

Last-Update: 2024-08-29
Forwarded: not-needed

The LICENSE in debian is contained in the copyright file that contains
the same information.

The files in share/doc/sos are already being populated into share/doc/sosreport
so we don't want them to be duplicated.
---
 setup.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/setup.py b/setup.py
index 6c0b09a1..c214570d 100644
--- a/setup.py
+++ b/setup.py
@@ -26,8 +26,6 @@ setup(
                             'man/en/sos-mask.1', 'man/en/sos-help.1',
                             'man/en/sos-upload.1']),
         ('share/man/man5', ['man/en/sos.conf.5']),
-        ('share/licenses/sos', ['LICENSE']),
-        ('share/doc/sos', ['AUTHORS', 'README.md']),
         ('config', ['sos.conf', 'tmpfiles/tmpfilesd-sos-rh.conf'])
     ],
     packages=find_packages(include=['sos', 'sos.*'])
